### DISCUSSION TITLE
No connection to 1434 port

### DISCUSSION THREAD

**Discussion by akrsnr** [dbeaver/dbeaver#13316] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version           windows 10
- DBeaver version                                                        21.1.3
- Additional extensions

#### Connection specification:
- Database name and version                                      sql server 2019
- Driver name                                                                com.microsoft.sqlserver.jdbc.SQLServerDriver
- Do you use tunnels or proxies (SSH, SOCKS, etc)?     no

It doesn't list schemas, dbs etc. Further, it even doesn't connect correctly. Something interrupts. I can connect nicely by SSMS but Dbeaver, I can also connect nicely to other dbs like mysql. 

```
SQL Error [08S01]: Connection reset ClientConnectionId:997643a9-d4ed-447e-9716-3fbc0c349be6
  Connection reset ClientConnectionId:997643a9-d4ed-447e-9716-3fbc0c349be6
  Connection reset ClientConnectionId:997643a9-d4ed-447e-9716-3fbc0c349be6
    Connection reset
    Connection reset

```

But when I activate 1433 port as seen on the last image by enabling TCP/IP,here is no problem. The issue is just to inform you.

![1](https://user-images.githubusercontent.com/17342831/126905927-3b1a0be7-b29b-4253-af57-3156bd735541.png)
![2](https://user-images.githubusercontent.com/17342831/126905929-6378e559-6946-4c45-ae57-917d177beafd.png)
![3](https://user-images.githubusercontent.com/17342831/126905932-8a7c03b4-f148-41f5-903d-54a1547c774c.png)
![4](https://user-images.githubusercontent.com/17342831/126905933-5fdbd646-11fb-4bb4-bfd4-35dcd2258a2f.png)
![5](https://user-images.githubusercontent.com/17342831/126905934-169dfff1-2057-463b-b642-d5d5b793bdd0.png)
![6](https://user-images.githubusercontent.com/17342831/126905936-8b176a73-fcb6-4ab5-96ec-3efec7a5cc57.png)
![7](https://user-images.githubusercontent.com/17342831/126906184-88c63082-553f-4c4c-b717-570c9d81493e.png)

**Comment by uslss** [comment, marked as answer]
It is expected behaviour. DBeaver uses jdbc driver that needs tcp/ip connection enabled to connect to db. SMSS uses another protocol.
